### PR TITLE
Add auction bot module

### PR DESCRIPTION
**Pull request by boodyvo**
- Updated the `auction-bot` and as a result `signing` to use cometbft, cossmos-sdk v0.47.10 and kava v0.26.1.
- Added simple CI for the repo
- Added integration test that was part of signer/testing into CI
- Outdate log for graphana in `spammer` switched to already used logrus (just updated imports)
- Removed go.work as it causes a lot of problems with dependencies management among modules (some of them require different versions of kava/tendermint/cometbft, etc) and there is no specific reason to keep go.work.

### CODE REVIEW

File: auction-bot/go.mod [hunk 1]
```diff
@@ -0,0 +1,239 @@
+module github.com/kava-labs/go-tools/auction-bot
+
+go 1.21.9
+
+require (
+	github.com/alexliesenfeld/health v0.8.0
+	github.com/cosmos/cosmos-sdk v0.47.10
+	github.com/go-chi/chi/v5 v5.0.7
+	github.com/joho/godotenv v1.5.1
+	github.com/kava-labs/go-tools/signing v0.0.0-20240823151227-93f98f6d066b
```
Comment (drklee3) [inline auction-bot/go.mod:10]: Thoughts on add a replace directive for signing so it always uses the latest signing version? Unless there is a need to pin to a specific version

```
replace (
	github.com/kava-labs/go-tools/signing => ../signing/
)
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        working-directory: ./kava/tests/e2e/kvtool
+
+      - name: Run kava docker container
+        run: KAVA_TAG=local kvtool t bootstrap
+
+      - name: Wait until kava node is ready to serve traffic
+        run: bash ${GITHUB_WORKSPACE}/go-tools/.github/scripts/wait-for-node-init.sh
+
+      - name: Run integration tests
+        run: KAVA_RPC_URL=http://localhost:26657 NETWORK=kava-local PORT=4000 SKIP_LIVE_NODE_TESTS=true make test-integration
```
Comment (drklee3) [inline .github/workflows/ci.yml:109]: Are these env vars used by the integration tests?

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+vet:
+	go vet ./...
+
+.PHONY: test
+test:
+	go test -v ./...
+
+.PHONY: test-integration
+test-integration:
+	cd signing/testing && go test -v -tags=integration -count=1
```
Comment (drklee3) [inline Makefile:38]: Should this be moved to the signing specific Makefile?

Comment (boodyvo) [reply Makefile:38]: Not sure if it make sense. If we will extend integration test to other modules it's easier to run one integration test for all of them with just `make test-integration`, than running it directory b directory, so would prefer to keep one common make command in the root

File: .github/workflows/ci.yml (hunk 2 above)
Comment (boodyvo) [reply .github/workflows/ci.yml]: Removed all extra vars

**Review by drklee3 — APPROVED** [review]
nice addition to ci

**Pull request merged**